packages/form-js-viewer/src/render/components/form-fields/Number.js

```diff
@@ -65,6 +63,11 @@ Number.create = function(options = {}) {
   };
 };
 
+Number.sanitizeValue = ({ value }) => {
+  const parsedValue = parseInt(value, 10);
+  return isNaN(parsedValue) ? null : parsedValue;
+};
```

Comment:
I hope we allow non-integer values some day in the future 🤡

Reply:
https://github.com/bpmn-io/form-js/issues/285

We certainly will